pkg/kclient/operators.go

```diff
@@ -211,6 +211,21 @@ func (c *Client) GetGVKFromGVR(gvr schema.GroupVersionResource) (schema.GroupVer
 	return mapper.KindFor(gvr)
 }
 
+func (c *Client) GetGVRFromGVK(gvk schema.GroupVersionKind) (schema.GroupVersionResource, error) {
+	cfg := c.GetClientConfig()
+
+	dc, err := discovery.NewDiscoveryClientForConfig(cfg)
```

Comment:
Do you think we could reuse `c.discoverClient` here instead of creating a new client? Should save some API calls, I guess.

Reply:
Oh, you are right. I also added the restmapper to the client so it is created only once.